index.js

```diff
@@ -64,7 +64,10 @@ function formatAttrs(attributes, opts) {
     if (!value && booleanAttributes[key]) {
       output += key;
     } else {
-      output += key + '="' + (opts.decodeEntities ? entities.encodeXML(value) : value) + '"';
+      output += key;
+      if (value || opts.xmlMode) {
```

Comment:
What is the type of `value`? I'm wary of applying `||` to non-Boolean operands, as it's a common source of bugs. Could `value` be `false` or `0`? If so, let's use a stricter check such as `value != null`.